src/AppBundle/Tests/Controller/DefaultControllerTest.php

```diff
@@ -12,6 +12,7 @@ public function testIndex()
 
         $crawler = $client->request('GET', '/');
 
-        $this->assertTrue($crawler->filter('html:contains("Homepage")')->count() > 0);
+        $this->assertEquals(200, $client->getResponse()->getStatusCode());
+        $this->assertContains('Welcome to Symfony', $client->getResponse()->getContent());
```

Comment:
I suggest keeping a crawler-based example too in the test (or remove the unused ``$crawler`` variable)